webplugin/js/app/kommunicate-ui.js

```diff
@@ -348,6 +308,29 @@ KommunicateUI={
         }
     });
 },
+searchFaqs:function(data){
+KommunicateKB.searchFaqs({
+    data:
+        { appId: data.appId, query: document.getElementById("km-faq-search-input").value }
+    , success: function (response) {
+        if (response.data && response.data.length === 0 && $applozic(".km-no-results-found-container").hasClass("n-vis")) {
+            $applozic(".km-no-results-found-container").addClass("vis").removeClass("n-vis");
+            $applozic(".km-talk-to-human-div p").text("We are here to help.");
+            $applozic(".km-talk-to-human-div").addClass("vis").removeClass("n-vis");
+        } else {
+            $applozic(".km-no-results-found-container").addClass("n-vis").removeClass("vis");
+            $applozic(".km-talk-to-human-div p").text(MCK_LABELS['looking.for.something.else']);
+            $applozic(".km-talk-to-human-div").addClass("vis").removeClass("n-vis");
+        }
+        
+        $applozic('#km-faq-list-container').empty();
+        $applozic.each(response.data, function (i, faq) {
+            $applozic("#km-faq-list-container").append('<li class="km-faq-list"  data-articleId="' + faq.id + '"><a class="km-faqdisplay"> <div class="km-faqimage">' + KommunicateUI.faqSVGImage + '</div><div class="km-faqanchor">' + faq.name + '</div></a></li>');
```

Comment:
This code can easily be written in javascript, please change it to a language with no dependency from external sources and make sure that new sidebox features are written in plain javascript and it will help us remove jquery from the sidebox code in the long run.